app/frontend/src/components/EditLocationMap.tsx

```diff
@@ -363,6 +364,9 @@ function RadiusSlider({ commit, initialRadius, redrawMap }: RadiusSliderProps) {
   const [radius, setRadius] = useState(initialRadius);
   return (
     <>
+      <Typography color="secondary" gutterBottom={true}>
```

Comment:
```suggestion
      <Typography variant="body2" gutterBottom>
```
I think that's sufficient